### PR TITLE
#152822394 - DocTool rule ** - Implement R - 06998 - Setting Zippy Ind.

### PR DESCRIPTION
**Pull request by JGovindaraj**

### CODE REVIEW

File: src/main/java/gov/ca/cwds/rest/api/domain/cms/Client.java [hunk 1]
```diff
@@ -1,23 +1,18 @@
 package gov.ca.cwds.rest.api.domain.cms;
```
Comment (tparkerosi) [inline src/main/java/gov/ca/cwds/rest/api/domain/cms/Client.java]: Needs to be tested with a specific Rules Test

File: src/main/java/gov/ca/cwds/rest/api/domain/cms/Referral.java [hunk 2]
```diff
@@ -1,28 +1,24 @@
 package gov.ca.cwds.rest.api.domain.cms;
```
Comment (tparkerosi) [inline src/main/java/gov/ca/cwds/rest/api/domain/cms/Referral.java]: Needs to be tested with a specific Rules Test

File: src/main/java/gov/ca/cwds/rest/business/rules/R06998ZippyIndicator.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package gov.ca.cwds.rest.business.rules;
+
```
Comment (tparkerosi) [inline src/main/java/gov/ca/cwds/rest/business/rules/R06998ZippyIndicator.java]: Add test for Yes and No. Ok to not test getter and setter

File: src/main/java/gov/ca/cwds/rest/api/domain/cms/Client.java (hunk 1 above)
Comment (JGovindaraj) [reply src/main/java/gov/ca/cwds/rest/api/domain/cms/Client.java]: added Test case.

File: src/main/java/gov/ca/cwds/rest/api/domain/cms/Referral.java (hunk 2 above)
Comment (JGovindaraj) [reply src/main/java/gov/ca/cwds/rest/api/domain/cms/Referral.java]: added test case

File: src/main/java/gov/ca/cwds/rest/business/rules/R06998ZippyIndicator.java (hunk 3 above)
Comment (JGovindaraj) [reply src/main/java/gov/ca/cwds/rest/business/rules/R06998ZippyIndicator.java]: added Test case

**Pull request merged**